crates/types/src/services/p2p.rs

```diff
@@ -109,9 +109,9 @@ pub struct DelegatePreConfirmationKey {
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 pub struct Preconfirmation {
     /// The ID of the transaction that is being pre-confirmed
-    tx_id: TxId,
+    pub tx_id: TxId,
     /// The status of the transaction that is being pre-confirmed
-    status: PreconfirmationStatus,
+    pub status: PreconfirmationStatus,
```

Comment:
If we are sending only this status, how the user will be able to get the receipts etc on the API ? Because this only contains block-height

Reply:
Not that I'm aware of, but I've heard others mention the receipts so maybe I'm missing part of the picture.